### PR TITLE
[RORDEV-75] SearchTemplateRequest support

### PR DESCRIPTION
**Pull request by clutroth**

### CODE REVIEW

**Comment by sscarduzio** [comment]
Hi @clutroth thanks for the PR! Please name this PR with a changelog-style description of the feature/fix. :)

**Comment by clutroth** [comment]
@pondzix

File: tests-utils/src/main/java/tech/beshu/ror/utils/elasticsearch/SearchManagerJ.java [hunk 1]
```diff
@@ -51,18 +51,33 @@ public SearchResult search(String endpoint) {
     return call(createSearchRequest(endpoint), SearchResult::new);
   }
 
+  public SearchResult storeScript(String scriptId, String query) { return call(createStoreScriptRequestRequest(scriptId, query), SearchResult::new); }
```
Comment (coutoPL) [inline tests-utils/src/main/java/tech/beshu/ror/utils/elasticsearch/SearchManagerJ.java]: do we need this method here? I see you've created ScriptManager?

Comment (clutroth) [reply tests-utils/src/main/java/tech/beshu/ror/utils/elasticsearch/SearchManagerJ.java]: Nope.

**Comment by coutoPL** [comment]
LGTM. Thanks

**Pull request merged**